elements/rh-demo/rh-demo.ts

```diff
@@ -47,16 +47,16 @@ export class RhDemo extends LitElement {
     return html`
       <div class="rh-demo-preview-area"></div>
       <div part="code" class="rh-demo-code-area">
-        <pfe-accordion>
-          <pfe-accordion-header>
+        <rh-accordion>
```

Comment:
we need to remove the dynamic `pfe-accordion` import and replace it with a static rh-accordion import